pom.xml

```diff
@@ -216,6 +222,12 @@
 		</dependency>
 
 		<dependency>
+			<groupId>org.apache.httpcomponents</groupId>
```

Comment:
So at least this should be a `runtime` dependency as it seems we have no compile time dependency on HC.